Golem.Tests/GolemTests.cs

```diff
@@ -59,22 +59,31 @@ public async Task Start_ChangeWallet_VerifyStatusAsync()
             Console.WriteLine("Path: " + golemPath);
 
             var golem = await TestUtils.Golem(golemPath, loggerFactory);
-            GolemStatus status = GolemStatus.Off;
-
-            Action<GolemStatus> updateStatus = (v) =>
+            try
             {
-                status = v;
-            };
 
-            golem.PropertyChanged += new PropertyChangedHandler<Golem, GolemStatus>(nameof(IGolem.Status), updateStatus, loggerFactory).Subscribe();
+                GolemStatus status = GolemStatus.Off;
+
+                Action<GolemStatus> updateStatus = (v) =>
+                {
+                    status = v;
+                };
+
+                golem.PropertyChanged += new PropertyChangedHandler<Golem, GolemStatus>(nameof(IGolem.Status), updateStatus, loggerFactory).Subscribe();
+
+                await golem.Start();
 
-            await golem.Start();
+                golem.WalletAddress = "0x1234567890123456789012345678901234567890";
 
-            golem.WalletAddress = "0x1234567890123456789012345678901234567890";
+                await golem.Stop();
 
-            await golem.Stop();
+                Assert.Equal(GolemStatus.Off, status);
 
-            Assert.Equal(GolemStatus.Off, status);
+            }
+            finally
+            {
+                await golem.Stop();
```

Comment:
Final `await golem.Stop()` for every test spawning Golem (test cases are not process isolated, and `yagna` process may remain alive after assertion error)

Reply:
Removed.